core/runtime/v2/bridge.go

```diff
@@ -26,30 +26,28 @@ import (
 
 	v2 "github.com/containerd/containerd/api/runtime/task/v2"
 	v3 "github.com/containerd/containerd/api/runtime/task/v3"
-
-	api "github.com/containerd/containerd/api/runtime/task/v3" // Current version used by TaskServiceClient
```

Comment:
I had a quick peek at the file in `main`, and it looks like there's 4 uses of `v3`, but 82 uses of the `api` alias.

Perhaps instead we should remove the `v3` alias; it would result in less code-churn, and also because the `api` alias describes why that version is picked.

Reply:
+1. also then later when we have `v4`, we can just change this `api -> v3` and use `api` alias for v4, without needing to change (most of) the `api` references.

Reply:
Updated the v3 to api alias